var/spack/repos/builtin/packages/speexdsp/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Speexdsp(AutotoolsPackage):
+    """SpeexDSP is a patent-free, Open Source/Free Software DSP library."""
+
+    homepage = "https://github.com/xiph/speexdsp"
+    url      = "https://github.com/xiph/speexdsp/archive/SpeexDSP-1.2.0.tar.gz"
+
+    version('1.2.0', sha256='d7032f607e8913c019b190c2bccc36ea73fc36718ee38b5cdfc4e4c0a04ce9a4')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
+
+    depends_on('fftw-api')
+
+    patch('mkl.patch')
+
+    def autoreconf(self, spec, prefix):
+        autoreconf('--install', '--verbose', '--force')
+
+    def configure_args(self):
+        args = []
+
+        if 'intel-mkl' in self.spec:
```

Comment:
What about `intel-parallel-studio+mkl`? I think you just want to use `'mkl' in spec` and `spec['mkl']` below.

Reply:
As far as I can tell, intel-parallel-studio does not provide `fftw-api`.
```console
$ spack providers fftw-api
fftw-api:
fftw@2.1.5  fftw@3:  intel-mkl@2017:
```

Reply:
Also, I have to have `spec['blas'].libs` because if this is built as a dependency, and if openmpi is in the spec, it will pick up scalapack and that breaks the build. I guess that is a concretization issue but that is what I observed, and why I put a comment in this block.

Reply:
Should I add 
```
provides('fftw-api',   when='+mkl')
```
to the `iuntel-parallel-studio` package as part of this PR?

Reply:
Up to you. Intel-parallel-studio is a huge can of worms that I would rather not touch unless someone complains. Let's leave it as is for now.